src/hooks/useTasks.js

```diff
@@ -89,3 +92,33 @@ function getTaskName(phase) {
     return 'Confirm appeal'
   }
 }
+
+function getRoundStatus(round, currentPhase) {
```

Comment:
What do you think of renaming this to something like isAppealTaskOpen ? Since we are not actually returning the status of the round